### PR TITLE
add notice_date to display "received date" in view.

### PR DESCRIPTION
**Pull request by stemount**
before and after

messages image 2210547796

### CODE REVIEW

File: web/modules/features/par_enforcement_flows/src/Form/ParEnforcementRaiseNoticeDetailsForm.php [hunk 1]
```diff
@@ -152,7 +155,7 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
     $enforcementAction = \Drupal::entityManager()->getStorage('par_data_enforcement_notice')->create($enforcementNotice_data);
 
     if ($enforcementAction->save()) {
-      $this->deleteStore();
+//      $this->deleteStore();
```
Comment (stephenb01) [inline web/modules/features/par_enforcement_flows/src/Form/ParEnforcementRaiseNoticeDetailsForm.php]: Is this meant to be commented out?

File: web/modules/features/par_enforcement_flows/src/Form/ParEnforcementRaiseNoticeDetailsForm.php [hunk 2]
```diff
@@ -129,13 +129,16 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
...
+    $time = new \DateTime();
+
     $enforcementNotice_data = [
       'notice_type' => $this->getTempDataValue('enforcement_type'),
       'summary' => $this->getTempDataValue('action_summary'),
       'field_regulatory_function' => $this->getTempDataValue('regulatory_functions'),
       'field_enforcing_authority' => $this->getDefaultValues('par_data_authority_id', '', 'par_authority_selection'),
       'field_organisation' => $this->getDefaultValues('par_data_organisation_id', '', 'par_data_organisation_id'),
       'field_partnership' => $partnership->id(),
+      'notice_date' => $time->format("Y-m-d"),
```
Comment (stephenb01) [inline web/modules/features/par_enforcement_flows/src/Form/ParEnforcementRaiseNoticeDetailsForm.php]: Is this the right format? Storing date as a string? Seen it before but not sure if this is right.

Comment (stemount) [reply web/modules/features/par_enforcement_flows/src/Form/ParEnforcementRaiseNoticeDetailsForm.php]: oui c'est le bon moyen

**Comment by stemount** [comment]
good spot @stephenb01, addressed.

**Pull request merged**